### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by FurblandChannel** [ThioJoe/YT-Spam-Lists#133] [General]
Does the program already filter out these kinds of bots?
![0C2BEE9B-CAE8-41CF-8FCE-3E62BBA8C8FC](https://user-images.githubusercontent.com/78236024/177080501-ddf91bda-1a89-47cb-b0d3-e71d9e6dcd3d.jpeg)

**Comment by Firecul** [comment]
It should catch it, if not you can always ask for `jamesify85` to be added by using [the form here.](https://github.com/ThioJoe/YT-Spam-Lists/issues/new?assignees=&labels=scam-account&template=report_spam_accounts.yaml&title=Scam+Account+Submission)
